doc/internal/man3/ossl_provider_new.pod

```diff
@@ -55,57 +58,69 @@ ossl_provider_get_params, ossl_provider_query_operation
 
 =head1 DESCRIPTION
 
-C<OSSL_PROVIDER> is a type that holds all the necessary information
+I<OSSL_PROVIDER> is a type that holds all the necessary information
 to handle a provider, regardless of if it's built in to the
 application or the OpenSSL libraries, or if it's a loadable provider
 module.
-Instances of this type are commonly refered to as I<provider object>s.
+Instances of this type are commonly refered to as "provider objects".
 
-A I<provider object> is always stored in a set of I<provider object>s
+A provider object is always stored in a set of provider objects
 in the library context.
 
-I<provider object>s are reference counted.
+Provider objects are reference counted.
 
-I<provider object>s are initially inactive, i.e. they are only
-recorded in the store, but are not used.
+Provider objects are initially inactive, i.e. they are only recorded
+in the store, but are not used.
 They are activated with the first call to ossl_provider_activate(),
 and are inactivated when ossl_provider_free() has been called as many
 times as ossl_provider_activate() has.
 
 =head2 Functions
 
-ossl_provider_find() finds an existing I<provider object> in the
-I<provider object> store by C<name>.
-The I<provider object> it finds gets its reference count
-incremented.
-
-ossl_provider_new() creates a new I<provider object> and stores it in
-the I<provider object> store, unless there already is one there with
-the same name.
-The reference counter of a newly created I<provider object> will
-always be 2; one for being added to the store, and one for the
-returned reference.
-To indicate a built-in provider, the C<init_function> argument must
+ossl_provider_find() finds an existing provider object in the provider
+object store by I<name>. 
+The provider object it finds gets its reference count incremented.
+
+ossl_provider_new() creates a new provider object named I<name> and
+stores it in the provider object store, unless there already is one
+there with the same name.
+The reference counter of a newly created provider object will always
```

Comment:
reference _count_

Reply:
Someone will have to explain to me some day with "counter" isn't quite right.  Possibly like you'd explain it to a 7 year old :wink:

But okie

Reply:
Does this help?  The common use is "reference count"  Nobody ever writes "reference counter"

It's because the English comp sci folks made that the common term, even with "refcount" as an abbreviation.

Don't blame English, blame programmers.

Reply:
The reference _count_ is part of the public api contract, the reference _counter_ a private implementation detail.

Reply:
> Nobody ever writes "reference counter"

You just made me nobody, thank you :wink:

Reply:
@mspncp, that actually made sense!  That'll help me remember, thank you.